portfolio/src/main/webapp/script.js

```diff
@@ -71,4 +83,13 @@ function populateDom() {
   addRandomQuote();
 }
 
+/** Removes comments from Datastore. */
+function deleteComments() {
```

Comment:
I would add a confirmation dialog to this operation since it is quite destructive.

```javascript
/** Removes comments from Datastore. */
function deleteComments() {
  if(window.confirm("Are you sure you want to delete all comments?")) {
    ...
  }
}
```